### PR TITLE
8.1 se

### PR DESCRIPTION
**Pull request by alviteri**

### CODE REVIEW

**Comment by alviteri** [comment]
@luk1337

**Comment by alviteri** [comment]
@jerpelea  this is ok ....

File: sepolicy_platform/file_contexts [hunk 1]
```diff
@@ -24,6 +25,7 @@
 
 /dev/block/platform/soc/c0c4000\.sdhci/by-name/TA              u:object_r:ta_block_device:s0
 /dev/block/bootdevice/by-name/TA                               u:object_r:ta_block_device:s0
+/dev/block/mmcblk0p1                                           u:object_r:ta_block_device:s0
```
Comment (luk1337) [inline sepolicy_platform/file_contexts]: Are you sure this is necessary? Isn't removing `/dev/block/mmcblk0p1 u:object_r:sd_block_device:s0` enough?

**Comment by alviteri** [comment]
Since TA is symlinked to this it should be labeled

File: sepolicy_platform/file_contexts (hunk 1 above)
Comment (alviteri) [reply sepolicy_platform/file_contexts]: Have you tried removing it ? Because TA is symlinked to this ....

Comment (luk1337) [reply sepolicy_platform/file_contexts]: No I haven't tried it.

Comment (alviteri) [reply sepolicy_platform/file_contexts]: I can remember with old 8994 ... Where TA was very closed about this ... So better have all TA labeled

Comment (alviteri) [reply sepolicy_platform/file_contexts]: @luk1337 it need to be labeled or you will get denials for blk_file as block_device .....

**Pull request merged**